zk_toolbox/crates/zk_supervisor/src/commands/lint.rs

```diff
@@ -0,0 +1,194 @@
+use clap::{Parser, ValueEnum};
+use common::{cmd::Cmd, logger, spinner::Spinner};
+use config::EcosystemConfig;
+use strum::EnumIter;
+use xshell::{cmd, Shell};
+
+use crate::messages::{
+    msg_running_linter_for_extension_spinner, msg_running_linters_for_files,
+    MSG_LINT_CONFIG_PATH_ERR, MSG_RUNNING_CONTRACTS_LINTER_SPINNER,
+};
+
+const IGNORED_DIRS: [&str; 18] = [
+    "target",
+    "node_modules",
+    "volumes",
+    "build",
+    "dist",
+    ".git",
+    "generated",
+    "grafonnet-lib",
+    "prettier-config",
+    "lint-config",
+    "cache",
+    "artifacts",
+    "typechain",
+    "binaryen",
+    "system-contracts",
+    "artifacts-zk",
+    "cache-zk",
+    // Ignore directories with OZ and forge submodules.
+    "contracts/l1-contracts/lib",
+];
+
+const IGNORED_FILES: [&str; 4] = [
+    "KeysWithPlonkVerifier.sol",
+    "TokenInit.sol",
+    ".tslintrc.js",
+    ".prettierrc.js",
+];
+
+const CONFIG_PATH: &str = "etc/lint-config";
+
+#[derive(Debug, Parser)]
+pub struct LintArgs {
+    #[clap(long, short = 'c')]
+    pub check: bool,
+    #[clap(long, short = 'e')]
+    pub extensions: Vec<Extension>,
+}
+
+#[derive(Debug, ValueEnum, EnumIter, strum::Display, PartialEq, Eq, Clone)]
+pub enum Extension {
+    #[strum(serialize = "rs")]
```

Comment:
You could use `serialize_all=lowercase` https://docs.rs/strum/latest/strum/additional_attributes/index.html